bots/tests-invoke

```diff
@@ -172,7 +172,18 @@ class PullTask(object):
             if subprocess.call([ "git", "ls-tree", "-d", "HEAD:bots/"], stdout=DEVNULL) != 0:
                 sys.stderr.write("Checking out bots directory from master ...\n")
                 subprocess.check_call([ "git", "checkout", "--force", "origin/master", "--", "bots/" ])
-        except:
+
+                # The machine code is special copy it from master too
+                machine = os.path.join(BOTS, "machine")
+                for name in os.listdir(machine):
+                    path = os.path.join(machine, name)
+                    if os.path.islink(path):
+                        os.unlink(path)
+                        code = subprocess.check_output([ "git", "show", "master:test/common/{0}".format(name) ])
```

Comment:
This and the next two lines should be outdented a level? I think you always want to do them, and just clean up the symlink if it already exists.